### DISCUSSION TITLE
How to get locales list inside nuxt.config.js

### DISCUSSION THREAD

**Discussion by learntheropes** [nuxt-modules/i18n#1455] [Q&A]
How can I get the locales list in the `nuxt.config.js` file?
I'm trying with

```
const $i18n = require('@nuxtjs/i18n')
const locales = $i18n.locales
```

But it doesn't work

**Comment by rchl** [comment]
That doesn't quite make sense. Locales are based on what you specify in the module's configuration so you already have that information in `nuxt.config.js`.

**Comment by learntheropes** [comment]
How can I get them when I generate the sitemap routes?
`this.$i18n.locales` doesn't work as well.
